cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -44,6 +46,36 @@ pub struct ParsedQObject {
     pub others: Vec<Item>,
 }
 
+impl ParsedQObject {
+    /// Extract either invokables or methods from [syn::ImplItem]'s from each Impl block
+    ///
+    /// These will have come from a impl cxx_qt::QObject<T> block
+    fn parse_impl_items(&mut self, items: &[ImplItem]) -> Result<()> {
+        for item in items {
+            // Check if this item is a method
+            if let ImplItem::Method(method) = item {
+                // Determine if this method is an invokable
+                if let Some(index) = attribute_find_path(&method.attrs, &["invokable"]) {
+                    // Remove the invokable attribute
+                    //
+                    // TODO: in the future there may be additional arguments
+                    // in the invokable attribute that we need to parse
+                    // such as type conversions, so we may need a ParsedInvokable later.
```

Comment:
Does it really help to have a TODO comment for something we don't know if we will need?

Reply:
Done, but we will need it at some point :-)